docs/outlook/mobile-log-appointments.md

```diff
@@ -0,0 +1,553 @@
+              </Group>
+            </OfficeTab>
+          </ExtensionPoint>
+        </DesktopFormFactor>
+        <MobileFormFactor>
+          <FunctionFile resid="residFunctionFile"/>
+          <ExtensionPoint xsi:type="MobileLogEventAppointmentAttendee">
+            <Control xsi:type="MobileButton" id="appointmentReadFunctionButton">
+              <Label resid="residLabel"/>
+              <Icon>
+                <bt:Image size="25" scale="1" resid="icon-16"/>
+                <bt:Image size="25" scale="2" resid="icon-16"/>
+                <bt:Image size="25" scale="3" resid="icon-16"/>
+                <bt:Image size="32" scale="1" resid="icon-32"/>
+                <bt:Image size="32" scale="2" resid="icon-32"/>
+                <bt:Image size="32" scale="3" resid="icon-32"/>
+                <bt:Image size="48" scale="1" resid="icon-48"/>
+                <bt:Image size="48" scale="2" resid="icon-48"/>
+                <bt:Image size="48" scale="3" resid="icon-48"/>
+              </Icon>
+              <Action xsi:type="ExecuteFunction">
+                <FunctionName>logCRMEvent</FunctionName>
+              </Action>
+            </Control>
+          </ExtensionPoint>
+        </MobileFormFactor>
+      </Host>
+    </Hosts>
+    <Resources>
+      <bt:Images>
+        <bt:Image id="icon-16" DefaultValue="https://contoso.com/assets/icon-16.png"/>
+        <bt:Image id="icon-32" DefaultValue="https://contoso.com/assets/icon-32.png"/>
+        <bt:Image id="icon-48" DefaultValue="https://contoso.com/assets/icon-48.png"/>
+        <bt:Image id="icon-80" DefaultValue="https://contoso.com/assets/icon-80.png"/>
+      </bt:Images>
+      <bt:Urls>
+        <bt:Url id="residFunctionFile" DefaultValue="https://contoso.com/commands.html"/>
+      </bt:Urls>
+      <bt:ShortStrings>
+        <bt:String id="residDescription" DefaultValue="Log appointment notes and other details to Contoso CRM."/>
+        <bt:String id="residLabel" DefaultValue="Log to Contoso CRM"/>
+      </bt:ShortStrings>
+      <bt:LongStrings>
+        <bt:String id="residTooltip" DefaultValue="Log notes to Contoso CRM for this appointment."/>
+      </bt:LongStrings>
+    </Resources>
+  </VersionOverrides>
+</VersionOverrides>
+```
+
+> [!TIP]
+> To learn more about manifests for Outlook add-ins, see [Outlook add-in manifests](manifests.md) and [Add support for add-in commands for Outlook Mobile](add-mobile-support.md).
+
+### Implement capturing appointment notes
+
+In this section, learn how your add-in can extract appointment details when the user selects the **Log** button.
+
+1. From the same quick start project, open the file **./src/commands/commands.js** in your code editor.
+
+1. Replace the entire content of the **commands.js** file with the following JavaScript.
+
+    ```js
+    // Office is ready.
+    Office.onReady(function () {
+        // Add any initialization code here.
+      }
+    );
+
+    function logCRMEvent(event) {
```

Comment:
This code looks different from the doc at https://microsoft-my.sharepoint.com/:w:/r/personal/jatingupta_microsoft_com/_layouts/15/Doc.aspx?sourcedoc=%7B0895E3FF-26C8-4C9D-AB2E-0B8CD2A4FB60%7D&file=CRM%20on%20Mobile%20Documentation.docx&nav=eyJjIjozMjI1NDc1NTh9&action=default&mobileredirect=true&share=IQH_45UIyCadTKsuC4zSpPtgAfFtOM1lqvGpKMNg-7S5hYk

logCRMEvent just assigns the passed event object to event declared variable and calls `getEventProperties()` method